OHHTTPStubs/Sources/OHHTTPStubs+Mocktail.m

```diff
@@ -0,0 +1,150 @@
+    
+    if(bError){
+        *error = [NSError errorWithDomain:MocktailErrorDomain code:kErrorFileFormatInvalid userInfo:[NSDictionary dictionaryWithObject:[NSString stringWithFormat:@"File '%@' has invalid URL regular expression pattern: %@.", fileURL.absoluteString,  lines[1]] forKey:NSLocalizedDescriptionKey]];
+        return nil;
+    }
+    
+    NSInteger statusCode = [lines[2] integerValue];
+    
+    NSMutableDictionary *headers = @{@"Content-Type":lines[3]}.mutableCopy;
+    
+    // From line 5 to '\n\n', expect HTTP response headers.
+    NSRegularExpression *headerPattern = [NSRegularExpression regularExpressionWithPattern:@"^([^:]+):\\s+(.*)" options:0 error:&bError];
+    if(bError){
+        *error = [NSError errorWithDomain:MocktailErrorDomain code:kErrorFileInternalError userInfo:[NSDictionary dictionaryWithObject:[NSString stringWithFormat:@"Internal error while stubbing file '%@'.", fileURL.absoluteString] forKey:NSLocalizedDescriptionKey]];
+        return nil;
+    }
+    
+    for (NSUInteger line = 4; line < lines.count; line ++) {
+        NSString *headerLine = lines[line];
+        NSTextCheckingResult *match = [headerPattern firstMatchInString:headerLine options:0 range:NSMakeRange(0, headerLine.length)];
+        
+        if (match) {
+            NSString *key = [headerLine substringWithRange:[match rangeAtIndex:1]];
+            NSString *value = [headerLine substringWithRange:[match rangeAtIndex:2]];
+            headers[key] = value;
+        } else {
+            *error = [NSError errorWithDomain:MocktailErrorDomain code:kErrorFileHeaderInvalid userInfo:[NSDictionary dictionaryWithObject:[NSString stringWithFormat:@"File '%@' has invalid header: %@.", fileURL.absoluteString, headerLine] forKey:NSLocalizedDescriptionKey]];
+            return nil;
+        }
+    }
+    
+    return [OHHTTPStubs stubRequestsPassingTest:^BOOL(NSURLRequest *request) {
+        NSString *absoluteURL = [request.URL absoluteString];
+        NSString *method = request.HTTPMethod;
+        
+        if ([absoluteURLRegex numberOfMatchesInString:absoluteURL options:0 range:NSMakeRange(0, absoluteURL.length)] > 0) {
+            if ([methodRegex numberOfMatchesInString:method options:0 range:NSMakeRange(0, method.length)] > 0) {
+                return YES;
+            }
+        }
+        
+        return NO;
+    } withStubResponse:^OHHTTPStubsResponse*(NSURLRequest *request) {
+        OHHTTPStubsResponse *response = [OHHTTPStubsResponse responseWithFileAtPath:fileURL.path
+                                                                         statusCode:(int)statusCode headers:headers];
+        [response.inputStream setProperty:[NSNumber numberWithUnsignedLongLong:([headerMatter dataUsingEncoding:NSUTF8StringEncoding].length + 2)] forKey:NSStreamFileCurrentOffsetKey];
```

Comment:
Please use modern ObjC and the boxing expression syntax `@(x)` instead of `[NSNumber numberWithUnsignedLongLong:x]`.